cron.yaml

```diff
@@ -0,0 +1,4 @@
+cron:
+- description: daily 9am EST ingestion job
+  url: /
+  schedule: every day 13:00
```

Comment:
- [ ]  update this to do ingestion jobs once OpenAPI spec is done and Cloud Endpoints are all working.